### ISSUE TITLE
Google Drive File Stream cask always reports as out of date

### ISSUE THREAD

**Issue by CoskunSunali** [Homebrew/brew#5992]
# What you were trying to do (and why)

Check if I have any outdated casks.

# What happened (include command output)

`google-drive-file-stream` always reports as outdated.

<details>
  <summary>Command output</summary>
  <pre>
  
brew cask outdated --greedy
google-drive-file-stream (latest) != latest
  
  </pre>
</details>

# What you expected to happen

`latest == latest` but homebrew says `latest != latest`

# Step-by-step reproduction instructions (by running `brew` commands)

Install `google-drive-file-stream` and run `brew cask outdated --greedy`

**Comment by vitorgalvao** [comment]
From [the `man` page](https://github.com/Homebrew/brew/blob/b9db71a1ed9c71a848e16fd1de094eac486a47ca/Library/Homebrew/manpages/brew-cask.1.md) (emphasis added):

> `outdated` [--greedy] [--verbose|--quiet] [ <token> ... ]:
    Without token arguments, display all the installed Casks that have newer
    versions available in the tap; otherwise check only the tokens given
    in the command line.
    If `--greedy` is given then also include in the output the Casks having
    `auto_updates true` or `version :latest`. Otherwise they are skipped
    because there is no reliable way to know when updates are available for
    them.

In other words, this is working as designed. `lastest` is not a fixed version, but a note indicating upstream doesn’t give versioned downloads. To change that, you’d need to either find an [appcast](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) for the software in question, or ask the upstream developer to provide versioned download links.

**Comment by CoskunSunali** [comment]
@vitorgalvao Thank you for the detailed explanation! I was in hope of reporting the fact that google-drive-file-stream didn't report any valid version and so that its cask maintainer would fix it.

However, you are right that it doesn't seem to be an issue with Homebrew itself. My bad.

Where do you suggest I log this issue? It [homebrew-cask](https://github.com/Homebrew/homebrew-cask) project the correct place?

**Comment by vitorgalvao** [comment]
> that its cask maintainer would fix it.

There are no maintainers of specific casks. It would be impossible for us to keep up.

> Where do you suggest I log this issue?

Nowhere. Like I said, *this is working as intended*, there is no issue here. To make the cask versioned, either you find the appcast (if it exists; if we knew of it we would have already added it) or you find a versioned download (you’d likely need to ask Google).

Why are you using `--greedy` anyway? There’s a reason that it’s a flag and not the default. I’d be willing to bet that software auto-updates anyway.

**Comment by CoskunSunali** [comment]
> There are no maintainers of specific casks.

That explains. Thanks!
